docs/blog/rendered/ibis-version-6.1.0-release.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<p>I think it would be useful to show at least one outer join example (left, right, or full).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/6747/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>